content/en/real_user_monitoring/browser/tracking_user_actions.md

```diff
@@ -20,11 +20,23 @@ further_reading:
 ---
 
 
-Real User Monitoring (RUM) SDKs detect user interactions performed during a user journey. Set the `trackInteractions` [initialization parameter][1] to `true` to enable this feature.
+Real User Monitoring (RUM) Browser SDK automatically detects user interactions performed during a user journey. 
 
-**Note**: The `trackInteractions` initialization parameter enables the collection of user clicks in your application. **Sensitive and private data** contained on your pages may be included to identify the elements interacted with.
+The automatic collection of user actions provides insights into user behavior, without having to instrument every single click in your application. It will help you achieve the following objectives:
+* Understand the performance of key interactions (for example, a click on the "Add to cart" button)
+* Quantify feature adoption
+* Identify the steps that led to a specific browser error
 
-Once an interaction is detected, all new RUM events are attached to the ongoing action until it is considered finished. The action also benefits from its parent view attributes such as browser information, geolocation data, and [global context][2].
+You can extend the collection of user interactions by [sending your own custom actions](#custom-actions).
+
+**Note**: The `trackInteractions` initialization parameter enables the collection of user clicks in your application. **Sensitive and private data** contained on your pages may be included to identify the elements interacted with. You can control which information is sent to Datadog by [manually setting an action name](#declaring-a-name-for-click-actions), or by [implementing global scrubbing rules in the Browser SDK][1].
+
+## What interactions are being tracked?
+
+The Browser SDK automatically tracks clicks. A click action is created if the following conditions are met:
+* Activity is detected within 100ms of click being handled (activity being defined as ongoing network requests or DOM mutations)
+* The click does not lead to a new page being loaded, in which case the Browser SDK will generate a new RUM View event
+* A name can be computed for the action. [Learn more about action naming](#declaring-a-name-for-click-actions)
```

Comment:
This is not exactly true currently no?
I remember trying to add dd-action-name but events were still not tracked if it triggered a page change for example?

Reply:
I will add "If **all** the following conditions are met"